tests/test_tagging.py

```diff
@@ -0,0 +1,301 @@
+# This file is part of ctrl_oods
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asynctest
+import asyncio
+import logging
+import os
+import tempfile
+from pathlib import PurePath
+from shutil import copyfile
+import yaml
+import lsst.utils.tests
+from lsst.ctrl.oods.directoryScanner import DirectoryScanner
+from lsst.ctrl.oods.fileIngester import FileIngester
+from lsst.daf.butler import Butler
+from lsst.daf.butler.registry import CollectionType
```

Comment:
`asynctest` is a third-party module and should be put *after* standard library modules. Also, you should put a blank line between each group of imports (standard vs third-party vs local). Or you could just run `isort` (can be installed using conda/mamba) on the file to make it fix things for you. :smile:

Reply:
Fixed